shared/gh/css/gh.components.css

```diff
@@ -733,7 +733,7 @@ tbody tr:last-child > td.fc-widget-content {
 
 .list-group .list-group-item .gh-series-borrowed {
     height: 100%;
-    width: 30px;
+    width: 40px;
 }
```

Comment:
`padding-left: 5px;` will be needed here as well, to get the link icons aligned with the arrows of the non-borrowed series.